mail-client/bower/bower-0.10.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit multiprocessing
+
+DESCRIPTION="A curses terminal client for the Notmuch email system"
+HOMEPAGE="https://github.com/wangp/bower"
+SRC_URI="https://github.com/wangp/bower/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+DEPEND="app-crypt/gpgme:=
+	>=dev-lang/mercury-11.07
+	sys-libs/ncurses:0="
+RDEPEND="app-crypt/gpgme:=
```

Comment:
Surely, you are missing `${DEPEND}` in here, otherwise this does not seem to make sense.

Reply:
this is an obvious mistake.

Reply:
(After counting slowly to ten) What is "this"? Please explain in detail what you consider a mistake. If (!) you are wondering why DEPEND is not a subset of RDEPEND, it is because of the way Mercury is used. This has been discussed with Michał Górny in the earlier PR I linked.

Reply:
Okay, on second view:
- `dev-lang/mercury` as a `DEPEND`-only is fine, nothing special here.
- Rather than duplicating `app-crypt/gpgme:=` and `sys-libs/ncurses:0=`, add them to `COMMON_DEPEND` and make both `DEPEND` and `RDEPEND` inherit it.